src/server/logout/prerequisites/provide-authed-user.js

```diff
@@ -5,6 +5,8 @@ import { getUserSession } from '~/src/server/common/helpers/auth/get-user-sessio
  * @type {import('@hapi/hapi').RouteOptionsPreObject}
  */
 const provideAuthedUser = {
+  // @fixme: code coverage
+  /* v8 ignore next */
```

Comment:
Discussed with @loveboat, this requires a different approach to testing which can be dealt with at a later stage